gfx_imgui.cpp

```diff
@@ -24,11 +24,15 @@ SOFTWARE.
 
 #include "gfx_imgui.h"
 
+#ifdef GFX_IMGUI_SOURCE
```

Comment:
Why is this new condition needed?

Reply:
Difference between pulling it in via package management vs directly form the repo. Package managers will provide a lib whereas the source repo doesnt define a lib target and so gets pulled in via source